src/components/DpDatepicker/DpDatepicker.vue

```diff
@@ -142,12 +148,22 @@ export default {
   },
 
   methods: {
+    addErrorFieldnameAttribute () {
+      const datePickerInput = document.getElementsByName(this.name)[0]
+      /**
+       * This attribute is needed for validation to display the field name in case of an error
+       * and must be set every time the Datepicker is mounted or updated.
+       */
```

Comment:
why does it need an update on every change?

Reply:
after updating the input value, the `data-dp-validate-error-fieldname` attribute that was added during mounting disappears from DOM, so I also [include it in the `emitUpdate` action](https://github.com/demos-europe/demosplan-ui/pull/841/files#diff-9749bd9f6c18eb9c907ecf005c35b25dc64bc5431f99f694aba87c0ee5c1c519R166)

Reply:
Thanks for the explanation